pkgs/tools/admin/awscli2/default.nix

```diff
@@ -72,6 +72,8 @@ with py.pkgs; buildPythonApplication rec {
     mkdir -p $out/share/zsh/site-functions
     mv $out/bin/aws_zsh_completer.sh $out/share/zsh/site-functions
     rm $out/bin/aws.cmd
+    PYTHONPATH=${src}:$PYTHONPATH
```

Comment:
```suggestion
    PYTHONPATH=${src}${python3.sitePackages}:$PYTHONPATH
```

Reply:
Hm. Actually I'm not sure this line is necessary at all.

My original hacky local tests for this looked something like:
```nix
  awscli2 = pkgs.awscli2.overrideAttrs(oldAttrs: rec {
    postBuild = ''
      PYTHONPATH=${oldAttrs.src}:$PYTHONPATH
      ${oldAttrs.src}/scripts/gen-ac-index --index-location $out/${python3.sitePackages}/awscli/data/ac.index
    '';
  });
```

but I couldn't get scripts/gen-ac-index to import from `/nix/store/1vna73hv36dkbw64l21r0a2hsq8gfxj1-source/awscli/autocomplete/local` without the `PYTHONPATH` line.

For some reason, though, it's not necessary in this PR - I'm not entirely sure why.